SEC/Equities/EquitiesExampleIndividuals.rdf

```diff
@@ -74,11 +82,126 @@
 	</owl:Ontology>
 	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AlphabetIncClassACommonStock">
+		<rdf:type rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+		<rdfs:label>Alphabet Inc. class A common stock</rdfs:label>
+		<skos:definition>common share that is identified as a class A series share in Alphabet Inc.</skos:definition>
+		<fibo-fbc-fi-fi:isLegallyRecordedIn rdf:resource="&fibo-be-ge-usj;StateOfCaliforniaJurisdiction"/>
+		<fibo-fbc-fi-fi:isNegotiable rdf:datatype="&xsd;boolean">true</fibo-fbc-fi-fi:isNegotiable>
+		<fibo-fnd-rel-rel:isIssuedBy rdf:resource="&fibo-sec-eq-eqind;AlphabetIncEquityIssuer"/>
+		<fibo-sec-sec-lst:hasHomeExchange rdf:resource="&fibo-fbc-fct-mkti;Exchange-XNAS"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AlphabetIncClassAFinancialInstrumentShortName">
+		<rdf:type rdf:resource="&fibo-sec-sec-iss;FinancialInstrumentShortName"/>
+		<rdfs:label>Alphabet Inc. class A financial instrument short name</rdfs:label>
+		<skos:definition>individual representing the Alphabet Inc. class A common share financial instrument short name (FISN)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;AlphabetIncClassACommonStock"/>
+		<fibo-sec-sec-iss:hasInstrumentDescription>SH CL A</fibo-sec-sec-iss:hasInstrumentDescription>
+		<fibo-sec-sec-iss:hasIssuerShortName>ALPHABET INC</fibo-sec-sec-iss:hasIssuerShortName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AlphabetIncClassCCapitalStock">
+		<rdf:type rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+		<rdfs:label>Alphabet Inc. class C capital stock</rdfs:label>
+		<skos:definition>common share that is identified as a class C series share in Alphabet Inc.</skos:definition>
+		<fibo-fbc-fi-fi:isLegallyRecordedIn rdf:resource="&fibo-be-ge-usj;StateOfCaliforniaJurisdiction"/>
+		<fibo-fbc-fi-fi:isNegotiable rdf:datatype="&xsd;boolean">true</fibo-fbc-fi-fi:isNegotiable>
+		<fibo-fnd-rel-rel:isIssuedBy rdf:resource="&fibo-sec-eq-eqind;AlphabetIncEquityIssuer"/>
+		<fibo-sec-sec-lst:hasHomeExchange rdf:resource="&fibo-fbc-fct-mkti;Exchange-XNAS"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AlphabetIncClassCFinancialInstrumentShortName">
+		<rdf:type rdf:resource="&fibo-sec-sec-iss;FinancialInstrumentShortName"/>
+		<rdfs:label>Alphabet Inc. class C financial instrument short name</rdfs:label>
+		<skos:definition>individual representing the Alphabet Inc. class C capital stock financial instrument short name (FISN)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;AlphabetIncClassCCapitalStock"/>
+		<fibo-sec-sec-iss:hasInstrumentDescription>SH CAP SH NV</fibo-sec-sec-iss:hasInstrumentDescription>
+		<fibo-sec-sec-iss:hasIssuerShortName>ALPHABET INC</fibo-sec-sec-iss:hasIssuerShortName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AlphabetIncEquityIssuer">
```

Comment:
To justify this being a separate individual, there needs to be at least 1 property value independent of the legal entity and the share.
The short name should be modeled as an identifier for the legal entity related to the FISN Scheme/Registry which is administered by ANNA.

Reply:
The FISN is modeled as a separate individual that includes both the issuer short name and instrument short name - so you are saying that you want two separate individuals?  

That means that for every issuer short name, which is a component of the FISN, I need a separate individual, and then also have the FISN, just to be clear?  The standard is only for the FISN, though.  And the issuer short name property is not unique without the description of the instrument as far as I can tell.

Reply:
The examples for FISN are fine (until we resolve the issue for composite identifiers, at which point we could consider the Short Name as a separate identifier for the Entity). My point here is that  FISN short name property is not a justification for the Issuer individual

Reply:
Ok - we disagree on that.  Issuer is a role, thus we need a separate individual to represent it, even if there were no properties on it.  An issuer is a party to a contract.  In this case, the issuer is the contract principal, although there may be cases for other kinds of instruments where the issuer and contract principal are switched, such as on some swap legs and option contracts.  The contract model is one case where everyone agreed that the more complex representation for parties was important. It becomes very complicated with options on swaps, and without party roles, impossible to model.

Reply:
There is nothing in FIBO that requires a role to have a separate individual as opposed to a class. If it's something specific to each "Contract" then that would require a separate Issuer individual for each Share - and the proposal has only one AlphabetIncIssuer reused for both Class A and Class C Alphabet Shares, which is not consistent with the justification. If the individual can be reused in this scenario for multiple contracts the individual for the business Entity can also be reused. If more sophisticated scenarios need an individual per contract then that can be represented as appropriate.

Reply:
There is a single set of issuance contract documents per class of instrument, so I disagree that it is one issuer to one share.  The issuer is the party to the issuance contract documents -and- a party to the individual shares, but there is no need to duplicate that issuer individual for each share, just as there is no need to represent every share as an individual.

Reply:
This is about the role of being an issuer - with respect to credit cards there is an additional identifier for the issuer in its role as an issuer - we've modeled that as well.  See IIN.  This is an example of an additional characteristic that is role-specific.  It only applies to credit / debit card issuers.

Reply:
Raised #873